include/cpuinfo_arm.h

```diff
@@ -15,6 +15,7 @@
 #ifndef CPU_FEATURES_INCLUDE_CPUINFO_ARM_H_
 #define CPU_FEATURES_INCLUDE_CPUINFO_ARM_H_
 
+#include <stdint.h>  // uint32_t
```

Comment:
this header is C99, is it ok for you ?
see: https://en.cppreference.com/w/c/types/integer

Reply:
Yes.
This is stated in the description of the project that we are expecting C99.

Reply:
FYI if we target cmake 3.1 (or above)
then we could add:
```cmake
set_target_properties(tgt PROPERTIES
  C_STANDARD 99
  C_STANDARD_REQUIRED ON)
```
src: https://cmake.org/cmake/help/v3.1/prop_tgt/C_STANDARD.html
https://cmake.org/cmake/help/v3.1/command/set_target_properties.html